phoenix-development/docker-compose.yml

```diff
@@ -26,6 +26,7 @@ services:
       HOST: "http://delta.dev.phoenixlan.no:3000"
       REACT_APP_OAUTH_CLIENT_ID: "phoenix-delta-dev"
       REACT_APP_MAIN_SITE: "http://dev.phoenixlan.no:3000"
+      STRIPE_PK: " "
```

Comment:
Wrong spot to place a STRIPE_PK. Look at the rest of the file. We use environment-files for secrets.

Reply:
Got it. Removed from the code and placed in a local environment file.